src/modify.jl

```diff
@@ -40,3 +40,26 @@ function rename(ta::TimeArray, colnames::Vector{Symbol})
 end
 
 rename(ta::TimeArray, colnames::Symbol) = rename(ta, [colnames])
+
+function rename(ta::TimeArray, d_colnames::Dict{Symbol, Symbol})
```

Comment:
I think we need to raise error, if there are duplicated values in the dictionary.
e.g. If user input this
```
Dict(:Open => :a, :Close => :a)
```

Reply:
hmm, I forgot that we will automatically change the duplicated column names.
Ignore this.

Reply:
Absolutely

```julia
julia> rename(ohlc, [:Open, :Open, :Low, :Close])
500×4 TimeArray{Float64,2,Date,Array{Float64,2}} 2000-01-03 to 2001-12-31
│            │ Open   │ Open_1 │ Low    │ Close  │
├────────────┼────────┼────────┼────────┼────────┤
│ 2000-01-03 │ 104.88 │ 112.5  │ 101.69 │ 111.94 │
```

even without duplicated values it could lead to duplicated coumn names and so this is fixed at an other layer.

```julia
julia> rename(ohlc, Dict(:High=>:Open))
500×4 TimeArray{Float64,2,Date,Array{Float64,2}} 2000-01-03 to 2001-12-31
│            │ Open   │ Open_1 │ Low    │ Close  │
├────────────┼────────┼────────┼────────┼────────┤
│ 2000-01-03 │ 104.88 │ 112.5  │ 101.69 │ 111.94 │
```